index.html

```diff
@@ -136,7 +137,7 @@ <h2>Facebook</h2>
           <li><button class="custom-buttonds" type="button">css</button></li>
           <li><button class="custom-buttonds" type="button">JavaScript</button></li>
         </ul>
-        <button class="big-buttonds" type="button">See Projects</button>
+        <button id="buttonds3" class="big-buttonds" type="button">See Projects</button>
       </div>
```

Comment:
- [ ] <b>REQUIRED:</b> The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.**